app/templates/admin/createSingleEvent.html

```diff
@@ -98,7 +98,7 @@ <h1 id="title">{{page_title}}</h1>
             <p>{{eventData.location}}</p>
           {%endif%}
         </div>
-        {% if isNewEvent %}
+        {% if isNewEvent and page_title != 'Create All Volunteer Training' %}
```

Comment:
Don't use the page title, use the event template tag. Because the page title could change and that would break this code, which we wouldn't expect

Reply:
I'm not sure if that template tag is here in the page, actually.